AWSSDKSwiftCLI/Sources/AWSSDKSwiftCLI/Resources/Package.Base.txt

```diff
@@ -267,13 +278,11 @@ private func target(_ service: String) -> Target {
             .smithyChecksums,
             .smithyWaitersAPI,
             .awsSDKCommon,
+            .awsSDKIdentityAPI,
             .awsSDKIdentity,
             .awsSDKHTTPAuth,
             .awsSDKEventStreamsAuth,
             .awsSDKChecksums,
-            "InternalAWSSTS",
-            "InternalAWSSSO",
-            "InternalAWSSSOOIDC",
```

Comment:
AWSSDKIdentityAPI is added to all service clients as a dependency.  The 3 internal clients are removed since they are not direct dependencies of the client (they are still linked though, as dependencies of AWSSDKIdentity.)